tests/integration/common-operator-integ-suite.sh

```diff
@@ -14,87 +14,141 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# To be able to run this script it's needed to pass the flag --ocp or --kind
-set -eu -o pipefail
+# To be able to run this script, it's needed to pass the flag --ocp or --kind
+set -eux -o pipefail
+
+check_arguments() {
+  if [ $# -eq 0 ]; then
+    echo "No arguments provided"
+    exit 1
+  fi
+}
 
-if [ $# -eq 0 ]; then
-  echo "No arguments provided"
-  exit 1
-fi
+parse_flags() {
+  while [ $# -gt 0 ]; do
+    case "$1" in
+      --ocp)
+        shift
+        OCP=true
+        ;;
+      --kind)
+        shift
+        OCP=false
+        ;;
+      *)
+        echo "Invalid flag: $1"
+        exit 1
+        ;;
+    esac
+  done
+
+  if [ "${OCP}" == "true" ]; then
+    echo "Running on OCP"
+  else
+    echo "Running on kind"
+  fi
+}
 
-while [ $# -gt 0 ]; do
-  case "$1" in
-    --ocp)
-      shift
-      OCP=true
-      ;;
-    --kind)
-      shift
-      OCP=false
-      ;;
-    *)
-      echo "Invalid flag: $1"
-      exit 1
-      ;;
-  esac
-done
-
-if [ "${OCP}" == "true" ]; then
-  echo "Running on OCP"
-else
-  echo "Running on kind"
-fi
+initialize_variables() {
+  WD=$(dirname "$0")
+  WD=$(cd "$WD" || exit; pwd)
 
-WD=$(dirname "$0")
-WD=$(cd "$WD" || exit; pwd)
+  NAMESPACE="${NAMESPACE:-istio-operator}"
+  DEPLOYMENT_NAME="${DEPLOYMENT_NAME:-istio-operator}"
+  CONTROL_PLANE_NS="${CONTROL_PLANE_NS:-istio-system}"
+  COMMAND="kubectl"
 
-NAMESPACE="${NAMESPACE:-istio-operator}"
-DEPLOYMENT_NAME="${DEPLOYMENT_NAME:-istio-operator}"
-CONTROL_PLANE_NS="${CONTROL_PLANE_NS:-istio-system}"
-COMMAND="kubectl"
+  if [ "${OCP}" == "true" ]; then
+    COMMAND="oc"
+  fi
 
-if [ "${OCP}" == "true" ]; then
-  COMMAND="oc"
-fi
+  echo "Using command: ${COMMAND}"
 
-BRANCH="${BRANCH:-maistra-3.0}"
+  BRANCH="${BRANCH:-maistra-3.0}"
```

Comment:
Is this variable used anywhere?

Reply:
The only place where I see this variable used is in the target bundle-publish, which we do not use for testing. Deleting